http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -384,6 +384,8 @@ object ExtensionType extends Enum[ExtensionType] {
   case object LabExtension extends ExtensionType
 }
 
+case class TraceId(uuid: UUID)
```

Comment:
just curious, why does using a separate data type fixes the null pointer exception?

Reply:
I think because it's not a value object

Reply:
Compare with https://github.com/broadinstitute/workbench-libs/blob/develop/model/src/main/scala/org/broadinstitute/dsde/workbench/model/models.scala#L6

Reply:
sorry forgot to comment back here...I saw the other comment :)